LenovoLegionToolkit.WPF/Controls/ColorCardControl.cs

```diff
@@ -196,6 +249,36 @@ private void ColorsPanel_MouseUp(object sender, MouseButtonEventArgs e)
             OnChanged?.Invoke(this, EventArgs.Empty);
         }
 
+        private void SystemAccentColor_Checked(object sender, RoutedEventArgs e)
+        {
+            Color systemAccentColor = ((SolidColorBrush)SystemParameters.WindowGlassBrush).Color;
```

Comment:
Does this work on Windows 10 and Windows 11?

Reply:
I'm using Windows 11 on all my devices. Don't have a Windows 10 to test.